pkg/v3/coordinator/log_event_coordinator.go

```diff
@@ -102,6 +125,10 @@ func (rc *reportCoordinator) checkEvents(ctx context.Context) error {
 			rc.logger.Printf("Skipping perform log in transaction %s as confirmations (%d) is less than min confirmations (%d)", evt.TransactionHash, evt.Confirmations, rc.minConfs)
 			continue
 		}
+		if evt.Type == ocr2keepers.StaleReportEvent {
+			rc.logger.Printf("Skipping perform log for upkeep %s in transaction %s as it is stale", evt.ID, evt.TransactionHash)
+			continue
+		}
```

Comment:
Instead of skipping the stale log, we should add it into the cache to ensure the id continues getting blocked in filters.